jupyter-notebooks/Data-API/search_and_preview_quickstart.ipynb

```diff
@@ -0,0 +1,496 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "50e6cd5c-78c4-48fc-a9e6-f0430217a51e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# For demo purposes, just grab the first image ID\n",
+    "id0 = image_ids[0]\n",
+    "id0_url = 'https://api.planet.com/data/v1/item-types/{}/items/{}/assets'.format(item_type, id0)\n",
+    "\n",
+    "# Returns JSON metadata for assets in this ID. Learn more: planet.com/docs/reference/data-api/items-assets/#asset\n",
+    "result = \\\n",
+    "  requests.get(\n",
+    "    id0_url,\n",
+    "    auth=HTTPBasicAuth(API_KEY, '')\n",
+    "  )\n",
+    "\n",
+    "# List of asset types available for this particular satellite image\n",
+    "print(result.json().keys())\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64156628-28d3-4a27-97a1-1a297cc1a54b",
+   "metadata": {},
+   "source": [
+    "## Rendering Previews On a Map"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a31b94a-9db1-4907-b0eb-dab11436287e",
+   "metadata": {},
+   "source": [
+    "First let's make a folium map centered around our area of interest. We use shapely to convert our geojson to a shape and then extract the centroid coordinates."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e21ec620-8707-444e-a4a7-cac027dc9e3d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import folium\n",
+    "from shapely.geometry import shape\n",
+    "\n",
+    "# Convert GeoJSON polygon to shapely geometry to calculate bounds/centroid\n",
+    "polygon = shape(geojson_geometry)\n",
+    "bounds = polygon.bounds\n",
+    "centroid = polygon.centroid\n",
+    "\n",
+    "# Calculate center and zoom based on bounds or centroid\n",
+    "map_center = [centroid.y, centroid.x]\n",
+    "print(f\"Map Center (based on centroid): {map_center}\")\n",
+    "\n",
+    "# Initialize the Folium map centered on the centroid of the polygon\n",
+    "folium_map = folium.Map(location=map_center, zoom_start=12)\n",
+    "\n",
+    "# Optional: add the GeoJSON polygon to the map for visualization\n",
+    "style_function = lambda feature: {\n",
+    "    'fillOpacity': 0,  # Transparent fill\n",
+    "    'color': 'black',  # Black border\n",
+    "    'weight': 3,       # Thickness of 3 pixels\n",
+    "    'dashArray': '5, 5'  # Dashed border\n",
+    "}\n",
+    "folium.GeoJson(geojson_geometry, style_function=style_function, name=\"Search Area\").add_to(folium_map)\n",
+    "\n",
+    "folium_map"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51391381-8941-42f0-9bfa-8fc3de8d481b",
+   "metadata": {},
+   "source": [
+    "Next, we will add preview scene tile layers to the map. To do this, we will leverage [Planet's Tile Services API](https://developers.planet.com/docs/basemaps/tile-services/#api-tile-service)"
```

Comment:
It might be worth indicating here that using Tile Services has ___ <no impact on quota?>

Reply:
Viewing tiles should charge some (small) amount of processing units (just like in Explorer), but it depends on the plan.

Reply:
I can add a comment here in some form, though

Reply:
We could definitely consider a different organization structure (e.g. using customer journey), it might make sense to do this as we consolidate Planet and SH more and unify things around "Gather" and "Analyze". If we do, it would be good to implement all the changes at once since it might impact lot's of links,.  Let's find some time to chat about it also with @olepinard who did the last re-organization.

Reply:
Great suggestion, Matt. I would be happy to join to shape and assist with some of that higher level re-organizing. It seems out of scope for this MR, so I will move ahead with the current organization as the "best approximate fit" for now.

Reply:
I agree that if a refactor is to be done, it should probably be done as a comprehensive refactor all at once, and not in this MR.